src/researchPage.js

```diff
@@ -1,11 +1,70 @@
-import { Wrapper, H1 } from "./components/styled";
+import { Wrapper } from "./components/styled";
 import { Link } from "react-router-dom";
 import Button from "./components/button/button";
+import Header from "./components/header/header";
+import ColorBlock from "./components/colorBlock/colorBlock";
 
 function ResearchPage() {
   return (
     <Wrapper>
-      <H1 bold>this is the research</H1>
+      <Header
+        text={"Color Theory"}
+        subtext={
+          "Learn more about the meaning behind each color palette and our research sources."
+        }
+      />
+      <ColorBlock
+        text={"relax, de-stress, calm, attention"}
+        string1={"#D9E5F3"}
+        string2={"#B2D3ED"}
+        string3={"#B2B5D9"}
+        string4={"#296CA0"}
+      />
+      <ColorBlock
+        text={"healing, pain relief, comfort"}
+        string1={"#E7F9C3"}
+        string2={"#ACDCD1"}
+        string3={"#97CD80"}
+        string4={"#719680"}
+      />
+      <ColorBlock
+        text={"joy, energy, positivity"}
+        string1={"#FFFFA2"}
+        string2={"#F6E7A4"}
+        string3={"#FEBB3D"}
+        string4={"#D46C00"}
+      />
+      <ColorBlock
+        text={"creativity, inspiration"}
+        string1={"#FDE8C0"}
+        string2={"#CFC8F8"}
+        string3={"#FCC663"}
+        string4={"#70669D"}
+      />
+      <ColorBlock
+        text={"strength, courage, focus"}
+        string1={"#FFA5A4"}
+        string2={"#D78F8E"}
+        string3={"#BABABA"}
+        string4={"#565656"}
+      />
+      <p>Sources</p>
+      <p>
+        Says, Deborah, et al. “Color Psychology: The Psychological Effects of
+        Colors.” Art Therapy, 24 Dec. 2012,
+        http://www.arttherapyblog.com/online/color-psychology-psychologica-effects-of-colors/#.YYLxU73MK58.{" "}
```

Comment:
can we add links to these articles? maybe make the wrapping `p` tag be an `a` tag whose `href` is the url that's in the source so that the user can click on the source to read the article